metabrainz/users/forms.py

```diff
@@ -12,8 +12,11 @@ class UserSignUpForm(Form):
     """
     contact_name = StringField(validators=[DataRequired("Contact name is required!")])
     contact_email = EmailField(validators=[DataRequired("Email address is required!")])
-    description = TextAreaField("How are you using our data?",
-                                validators=[DataRequired("Please, tell us how you (will) use our data.")])
+    usage_desc = TextAreaField("How are you using our data?",
+                               validators=[
+                                   DataRequired("Please, tell us how you (will) use our data."),
+                                   Length(max=150, message="Please, limit usage description to 150 characters.")
+                               ])
```

Comment:
Users shouldn't even get to seeing any of these warning if browser handles validation properly.